components/src/core/score-card/scorecard.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'pxb-scorecard',
+    template: `
+        <mat-card class="pxb-root">
+            <div class="pxb-header">
+                <div class="pxb-header-background"></div>
+                <div class="pxb-header-wrapper">
+                    <div class="pxb-header-text">
+                        <mat-card-title class="pxb-title">{{ headerTitle }}</mat-card-title>
+                        <mat-card-subtitle class="pxb-subtitle">{{ headerSubtitle }}</mat-card-subtitle>
+                        <mat-card-subtitle class="pxb-info">{{ headerInfo }}</mat-card-subtitle>
+                    </div>
+                    <div class="pxb-action-items">
+                        <ng-content select="[action-items]"></ng-content>
+                    </div>
+                </div>
+            </div>
+            <mat-card-content>
+                <div class="pxb-body">
+                    <ng-content select="[body]"></ng-content>
+                    <div class="pxb-badge-wrapper" [style.marginTop.px]="badgeOffset || 'inherit'">
+                        <ng-content select="[badge]"></ng-content>
+                    </div>
+                </div>
+                <mat-divider></mat-divider>
+                <div class="pxb-action-row">
+                    <ng-content select="[action-row]"></ng-content>
+                </div>
```

Comment:
there should not be a divider if no action-row is provided

Reply:
![image](https://user-images.githubusercontent.com/29152776/78354395-c849e500-7579-11ea-823a-7765faa87409.png)

Reply:
Good catch, I get to try out the conditional based on the presence of a `ng-content` being provided now.

Reply:
Updated this.